.github/workflows/github-actions.yml

```diff
@@ -69,9 +69,17 @@ jobs:
       run: |
         make verify
 
+      # This step only runs when the event type is a pull_request
     - name: Integrate Test
+      if: ${{ github.event_name == 'pull_request' }}
       run: |
-        chmod +x integrate_test.sh && ./integrate_test.sh ${{github.event.pull_request.head.repo.full_name}} ${{github.event.pull_request.head.sha}}
+        chmod +x integrate_test.sh && [[ -n "${{github.event.pull_request.head.repo.full_name}}" ]] && [[ -n "${{github.event.pull_request.head.sha}}" ]] && [[ -n "${{github.base_ref}}" ]] && ./integrate_test.sh ${{github.event.pull_request.head.repo.full_name}} ${{github.event.pull_request.head.sha}} ${{github.base_ref}}
+
+      # This step only runs when the event type is a push
+    - name: Integrate Test
```

Comment:
I guess change this name to 'Integrate Test With Dubbo-go-samples 1.5' would be better to distinguish two procedure

Reply:
Does it mean to distinguish pull_request from push?